src/sql/store.py

```diff
@@ -175,5 +178,48 @@ def _flatten(elements):
     return [element for sub in elements for element in sub]
 
 
+def store_snippet_as_sql(sql_command, snippet_name):
+    """
+    Store snippet as a SQl file
+
+    Parameters
+    ----------
+    command : str
+        query to be saved as the snippet .
+
+    saved_snippet : str
+        Name of the saved snippet
+    """
+
+    snippets_dir = "jupysql-snippets"
+    snippet_path = Path(snippets_dir) / f"{snippet_name}.sql"
+    snippet_path.parent.mkdir(parents=True, exist_ok=True)
+    with open(snippet_path, "w") as file:
+        file.write(sql_command)
+    message = (
+        """<div style="border: 2px solid #e8e8e8; padding: 10px; margin: 5px 0; """
+        """background-color: #f7f7f7; font-family: Arial, sans-serif;">"""
+        """<strong>Note:</strong> Manual editing of .sql files may not be """
+        """reflected when reopening the notebook. <br>Please edit snippets directly """
+        """ in the notebook to ensure consistency."""
+        """</div>"""
+    )
+
+    display(HTML(message))
```

Comment:
Why are you displaying html instead of `display.message`?

Reply:
changed this

Reply:
@edublancas What do you think if we create a unique style for all our "information" prints for the user? same as `display.success("success message")` we can do something like `display.information("some important information")`.

Reply:
So now the default style for the display.message is None, maybe we can set something else as the default: 
`def message(message, style=None):`

Reply:
> So now the default style for the display.message is None, maybe we can set something else as the default:
> `def message(message, style=None):`

Personally, I think it gives too much freedom and users might apply different styles to the message which might cause inconsistency and will be difficult to maintain (e.g. they should follow these styling rules `style="font-size: 12px; font-style: italic;"`)

I don't think it's very important for this PR so we can keep it as is and create a new issue to make a consistent styling for informative messages to the user. 

@edublancas what do you think?